ui/src/app/pages/admin/admin-workspace.tsx

```diff
@@ -39,13 +61,76 @@ const PurpleLabel = ({style = {}, children}) => {
   </label>;
 };
 
+const workspaceInfoField = (labelText: string, divContents: React.ReactFragment) => {
```

Comment:
Existing issue, so feel free to ignore this for now:

I'm wondering whether it might be more idiomatic to just make this into a private function component as you did for FileDetailsTable, since it's only a few characters removed from that currently (maybe @petesantos has an opinion on this):

```
const WorkspaceInfoField = ({labelText, children}) => {
  ...
};
```
later..
```
<WorkspaceInfoField labelText="..."> ... </WorkspaceInfoField>
```

Reply:
done